docs/en/observability/synthetics-reference/lightweight-config/common.asciidoc

```diff
@@ -234,6 +234,62 @@ By default, monitors are automatically retested if the monitor goes from "up" to
 retest_on_failure: false
 ----
 
+////////////////////////
+locations
+////////////////////////
+| [[monitor-locations]] *`locations`*
+(list of https://github.com/elastic/synthetics/blob/{synthetics_version}/src/locations/public-locations.ts#L28-L37[`SyntheticsLocationsType`])
+a| Where to deploy the monitor. Monitors can be deployed in multiple locations so that you can detect differences in availability and response times across those locations.
+
+To list available locations you can:
+
+* Run the <<elastic-synthetics-locations-command,`elastic-synthetics locations` command>>.
+* Go to *Synthetics* -> *Management* and click *Create monitor*. Locations will be listed in _Locations_.
+
+// If no `locations` or `private_locations` are specified in the monitor configuration, but is defined in the <<synthetics-configuration,project configuration file>>, the monitors will be pushed to the locations listed in the project configuration file.
+
+*Examples*:
+
+[source,yaml]
+----
+locations: ["asia-northeast1-a", "asia-south1-a"]
```

Comment:
This would be similar to the public location list we have declared here - https://github.com/elastic/synthetics/blob/d787cff0232536152a918f7964ab4e3a49977c50/src/locations/public-locations.ts#L27-L38

Internally we map these to these long ids to make it easier for devs to configure locations.